themes/default/content/cloud-engineering-summit/_index.md

```diff
@@ -8,6 +8,49 @@ layout: cloud-engineering-summit
 hubspot_form_id: 15b92d63-158e-4fc0-8b2c-c4bf3fbfee27
 sessionize_id: wbl7wd7l
 
+workshops:
```

Comment:
Missing the NGINX one (it's in Sessionize but not scheduled yet.):
> Going from Zero to Production in Kubernetes with NGINX and Pulumi
>
>In this workshop, we will highlight the NGINX MARA project and show you how you can stand up an entire application infrastructure completely automated with Pulumi.  We will walk through all of the different components and tooling to build a Kubernetes based microservices application deployment, with all of the required tools in place leverage production ready best practices. 

Speakers:
* Damian Curry
* Elijah Zupancic

Workshop is on 10/21; not sure when exactly.

Reply:
It's actually been scheduled in sessionize and is on the agenda I believe

Reply:
👍